src/main/java/tempest/ui/ViewManager.java

```diff
@@ -45,8 +47,13 @@ public void changeView(String name) {
   public void changeToPrevious() {
     history.pop();
     String lastView = history.peek();
+    // Updates the delete module button
     if (!viewExists(lastView))
       System.err.println("Invalid State");
+    else if (lastView.equals("manageModulesPage")) {
```

Comment:
Now that the page names PR have been merged can you make sure that you've rebased?

Reply:
Just changed it to match.